### PR TITLE
Tools 175 gather geo metadaata pooled exp

### PR DESCRIPTION
**Pull request by jychien**
@jahilton I was able to gather the geo metadata for LATDS555LKZ. Let me know if there are any additional changes you think would be good to add or if there's anything that needs attention. I've summarized on [TOOLS ticket](https://lattice.atlassian.net/browse/TOOLS-175).

### CODE REVIEW

**Comment by jychien** [comment]
@jahilton I found a bug as it is not printing the md5sum for the RawMatrixFiles.

File: scripts/geo_metadata.py [hunk 1]
```diff
@@ -188,35 +195,35 @@ def gather_objects(input_object, start_type=None):
...
-				unfiltered_matrices.append(obj)
-		else:
-			continue
-	return my_raw_matrices, unfiltered_matrices
 
-
-def get_dataset(dataset):
-	global geo_study
-	results = lattice.get_object(dataset, connection)
+	field_lst = RAW_MATRIX+['@id','accession', 'libraries', 'derived_from', 's3_uri', 'status']
```
Comment (jahilton) [inline scripts/geo_metadata.py:207]: add md5sum here?

Comment (jychien) [reply scripts/geo_metadata.py:207]: Yes! That should be it. Thanks!

Comment (jychien) [reply scripts/geo_metadata.py:207]: Just tested it, and that was it. I'll push a commit to fix. Thanks!

**Comment by jychien** [comment]
@jahilton I've updated the ordering of columns. I was going to put the constants in a separate file (ie: constants.py). But then we need a constants.py for the flattener in the same directory, so left it as is.

**Review by jahilton — APPROVED** [review]
LGTM

**Pull request merged**